mex/common/backend_api/connector.py

```diff
@@ -45,20 +46,25 @@ def _set_url(self) -> None:
         settings = BaseSettings.get()
         self.url = urljoin(str(settings.backend_api_url), self.API_VERSION)
 
-    def fetch_extracted_items(
+    def fetch_extracted_items(  # noqa: PLR0913
         self,
-        query_string: str | None,
-        stable_target_id: str | None,
-        entity_type: list[str] | None,
-        skip: int,
-        limit: int,
+        *,
+        query_string: str | None = None,
+        stable_target_id: str | None = None,
+        entity_type: list[str] | None = None,
+        referenced_identifier: list[str] | None = None,
+        reference_field: str | None = None,
+        skip: int = 0,
+        limit: int = 100,
     ) -> PaginatedItemsContainer[AnyExtractedModel]:
         """Fetch extracted items that match the given set of filters.
 
         Args:
             query_string: Full-text search query
             stable_target_id: The item's stableTargetId
             entity_type: The item's entityType
+            referenced_identifier: Optional merged item identifiers filter
+            reference_field: Optional field name to filter for
             skip: How many items to skip for pagination
             limit: How many items to return in one page
```

Comment:
All arguments are optional. Can we either add "optional" to all or remove it for all args and assume that devs infer that args with a default are optional.